apps/core-app/src/main/modules/ai/providers/nexus-provider.ts

```
@@ -0,0 +1,235 @@
+}
+
+function parseJsonResult<T extends object>(value: unknown, fallback: T): T {
+  if (value && typeof value === 'object' && !Array.isArray(value)) {
+    return { ...fallback, ...(value as Partial<T>) }
+  }
+  if (typeof value !== 'string') {
+    return fallback
+  }
+
+  const trimmed = value.trim()
+  const fenced = /```(?:json)?\s*([\s\S]*?)```/i.exec(trimmed)?.[1]?.trim()
```

ESLint

[error] The quantifier '\s*' can exchange characters with '[\s\S]*?'. Using any string accepted by /\s+/i, this can be exploited to cause at least polynomial backtracking.

(regexp/no-super-linear-backtracking)